run.sh

```diff
@@ -47,7 +45,10 @@ fi
 if [ -z "$COLLECTD_FLUSHINTERVAL" ]; then
 	COLLECTD_FLUSHINTERVAL=$COLLECTD_INTERVAL
 fi
-
+if [ ! -S /var/run/docker.sock ]; then
+    echo "The docker socket was not mounted to the container, the collectd docker plugin will be disabled"
```

Comment:
"The Docker socket was not mounted into this container; the CollectD Docker plugin will be disabled."